src/rpcq.lisp

```diff
@@ -273,6 +273,34 @@ We distinguish between the following options for any field type:
     tbl))
 
 
+(defun snake-to-kebab (string)
```

Comment:
Check out `src/utilities.lisp` for some related routines.

Reply:
Good call. I moved the two helpers there, and updated `str->lisp-keyword` to use `snake-to-kebab`.